lib/scala/runtime-version-manager/src/main/scala/org/enso/runtimeversionmanager/components/GraalRuntime.scala

```diff
@@ -35,6 +35,12 @@ case class GraalRuntime(version: GraalVMVersion, path: Path) {
     javaHome / "bin" / executableName
   }
 
+  /** The path to the `gu` executable associated with this runtime. */
+  def guExecutable: Path = {
+    val executableName = if (OS.isWindows) "gu.cmd" else "gu"
+    javaHome / "bin" / executableName
+  }
+
```

Comment:
This seems quite brittle. We should probably actually look up the windows version on the system with an allowed set of extensions. Then you could have a generic function rather than duplicating what is essentially the same logic.